platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/initialization/ConfigurationCacheProblemsListener.kt

```diff
@@ -107,25 +107,25 @@ class DefaultConfigurationCacheProblemsListener internal constructor(
         // the build to fail when it really shouldn't.
         val contextTask = runningTask ?: task
         val isExecutingOtherTask = contextTask != task
-        problemsListenerFor(contextTask).onProblem(
+        problemsListenerFor(contextTask).onExecutionTimeProblem(
             problemFactory.problem {
                 if (isExecutingOtherTask) {
                     text("execution of task ")
                     reference(contextTask.path)
                     text(" caused invocation of ")
                     reference(invocationDescription)
-                    text(" in other task at execution time which is unsupported.")
+                    text(" in other task at execution time which is unsupported with the configuration cache.")
                 } else {
                     text("invocation of ")
                     reference(invocationDescription)
-                    text(" at execution time is unsupported.")
+                    text(" at execution time is unsupported with the configuration cache.")
                 }
             }
                 .exception(
                     if (isExecutingOtherTask) {
-                        "Execution of $runningTask caused invocation of '$invocationDescription' by $task at execution time which is unsupported."
+                        "Execution of $runningTask caused invocation of '$invocationDescription' by $task at execution time which is unsupported with the configuration cache."
                     } else {
-                        "Invocation of '$invocationDescription' by $task at execution time is unsupported."
+                        "Invocation of '$invocationDescription' by $task at execution time is unsupported with the configuration cache."
```

Comment:
ℹ️  We already mention CC in the same manner in all serialization-time problems. Now that the execution-time problems result in independent build failures, it's even more important to indicate that they are caused by CC being enabled.